test/Interpreter/SDK/object_literals.swift

```diff
@@ -20,7 +20,7 @@ LiteralsTestSuite.test("file") {
 }
 
 LiteralsTestSuite.test("image") {
-  let image = #imageLiteral(resourceName: NSImageNameComputer)
```

Comment:
I think it is worth turning this into a negative case, which has the error message, and making a new case with the positive.  Also, I'd be glad to see a test for this in both Swift 3 and Swift 4 modes, if it is a warning in Swift 3 mode.